doc/source/user_guide/io.rst

```diff
@@ -3924,6 +3924,20 @@ The look and feel of Excel worksheets created from pandas can be modified using
 * ``float_format`` : Format string for floating point numbers (default ``None``).
 * ``freeze_panes`` : A tuple of two integers representing the bottommost row and rightmost column to freeze. Each of these parameters is one-based, so (1, 1) will freeze the first row and first column (default ``None``).
 
+.. note::
+
+    As of Pandas 3.0, by default spreadsheets created with the ``to_excel`` method
+    will not contain any styling. Users wishing to bold text, add bordered styles,
+    etc in a worksheet output by ``to_excel`` can do so by using ``Styler.to_excel``
```

Comment:
Can you use `` :meth:`Styler.to_excel` `` and also add a reference to the section in the User Guide: https://pandas.pydata.org/docs/user_guide/style.html#Export-to-Excel